components/common/src/ome/api/RawPixelsStore.java

```diff
@@ -130,10 +130,13 @@
 
     public Object getResolutionDescriptions();
 
+    /* @see ome.io.nio.PixelBuffer#getResolutionLevels() */
```

Comment:
These links do not render in the published Javadoc but it might be because of the component dependency graph (`common` being upstream of `romio`)